packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -4817,6 +4838,18 @@ describe('StripeHelper', () => {
         assert.equal(thrownError.name, 'TypeError');
       });
 
+      it('throws an exception if invoice line items doesnt have type = "subscription"', async () => {
+        const fixture = deepCopy(invoicePaidSubscriptionCreate);
+        fixture.lines.data[0].type = 'invoiceitem';
+        try {
+          await stripeHelper.extractInvoiceDetailsForEmail(fixture);
+          assert.fail();
+        } catch (err) {
+          assert.isNotNull(err);
```

Comment:
If we're in a `catch` block, I think we can always assume the exception object is not `null`; perhaps replace this assertion with something that asserts we get the expected, descriptive error message?
```js
assert.equal(err.message, 'Could not find next invoice date for subscription.');
```

Reply:
Ah no, I wanted to include this change, but didn't realize that my branch push didn't go through. I'll include the fix in my next PR.